validators/faq.adoc

```diff
@@ -0,0 +1,155 @@
+
+http://skale.chat/[Ask a question]
+
+== Validator Node
+
+=== How do I change my Node's IP address?
+
+It would be best to avoid having to change your node's IP address. It's recommended to use a permanent IP that can dynamically point to a node's machine IP (for example, elastic IP) so you can update machines if needed without having to change the IP recorded in SKALE Manager during node registration.
+
+If you must change your node IP in SKALE Manager, you must perform a node exit and re-registration in the first 3 days (72 hours) of the month. Also, performing a node exit could be expensive to the node operator if it's currently supporting SKALE Chains.
+
+To perform a node exit, and register a new machine and IP:
+
+1. Execute `skale exit start` on the exiting node.
+2. Verify the exit process by executing `skale exit status`. (You don't need to wait for exit to finalize)
+3. Create a new node machine
+4. Link the new node to the validator ID.
+5. Register the node with under the new IP.
+
+To perform a node exit, keep the original node machine and its data and update the IP, follow these steps:
+
+1. Perform the general node exit procedure steps 1 and 2 above.
+2. Go to `~/.skale/node_data/node_config.json` and delete all key-value **except `sgx_key_name`**.
+3. Link the node to the validator ID again.
+4. Register node with a new IP.
+5. Execute `skale node update .env`.
+
+=== How should validators configure back up and restore?
+
+Validators are responsible for backing up everything on the node. SKALE Network is relying on validators to complete the backup procedure for each of the nodes. SKALE Network will be replicating the data for each SKALE Chain across 16 nodes, adding an additional layer of data availability. If a node goes down for a long time the other 2/3 of 15 will be responsible for taking the SKALE Chain's snapshot and shuffling the SKALE Chain in the SKALE Network. If a validator node is down for a long time, it may be reallocated to service new SKALE Chains depending on the length of the downtime.  
+
+Please check https://github.com/skalenetwork/skale-node-cli/blob/develop/README.md#node-commands[node back up] and https://github.com/skalenetwork/sgxwallet/blob/develop/docs/backup-procedure.md[sgx_backup] to learn more about how to back up node or SGXWallet, and restore.
+
+=== Where are the network communications between containers, shared configuration files and volumes stored?
+
+You can find these details in GitHub [skale-node](https://github.com/skalenetwork/skale-node) repository and [docker-compose.yml](https://github.com/skalenetwork/skale-node/blob/f928b95e69c548f12b4b21bd11a16fe2d239b83b/docker-compose.yml)  
+
+=== How do I access the SKALE Chain's IP and ports?
+
+Please perform the following in your terminal and pick one of the names shows up under "schains" folder:  
+
+```shell
```

Comment:
It's not quite correct. This path of the mounted directory inside container. To get this data from host you can check ~/.skale/node_data/schains/schain_schain_name.json file or use `skale schains show-rules` to show firewall rules info that contains IP and ports.

Reply:
+1, the path is not correct.

Also, config file could be retrieved using node-cli, without going into internal folders:
```bash
skale schains config SCHAIN_NAME
```

Reply:
done